docs/Algorithms.md

```diff
@@ -11,6 +11,7 @@ Each compression method receives its own hyperparameters that are organized as a
   - Mixed-bitwidth quantization
   - Automatic bitwidth assignment based on HAWQ
   - Automatic quantization parameter selection and activation quantizer setup based on HW config preset
+  - Automatic bitwidth assignment based on AutoQ, a Deep Reinforcement Learning algorithm to select best mixed precision given quality metric and HW type.
```

Comment:
Suggest changing `...on AutoQ, a Deep...` to `...on AutoQ - a Deep...`

Reply:
Pushed the following description, imo we should reference HAQ here too.

- Automatic bitwidth assignment mode AutoQ, based on HAQ, a Deep Reinforcement Learning algorithm to select best mixed precision given quality metric and HW type.